dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -167,8 +174,9 @@ def readRecord(self,reader,writer,file_type,interfaces,row_number,job_id,fields,
                 writer.write(["Formatting Error", ValidationError.readErrorMsg, str(row_number), ""])
                 error_list.recordRowError(job_id,self.filename,"Formatting Error",ValidationError.readError,row_number,severity_id=interfaces.validationDb.getRuleSeverityId("fatal"))
                 row_error_found = True
-            return {}, reduce_row, True, False, row_error_found
-        return record, reduce_row, False, False, row_error_found
+
+            return {}, reduce_row, True, False, row_error_found, None
+        return record, reduce_row, False, False, row_error_found, flex_cols
```

Comment:
Given how many args we're returning, would it make sense to define a `namedtuple` to make this a bit easier to read? Relatedly, would you mind modifying the docstring to explain what the return values are? Finally, is there any reason to return `None` for the flex_cols but `{}` for the record? Seems like we should do one or the other but not both.